Dockerfile

```diff
@@ -15,11 +15,15 @@ RUN dnf -y update && \
         wget \
         git \
         python36 \
-        java-1.8.0-openjdk-devel \
+        java-11-openjdk-devel \
         maven \
         cargo \
     && dnf clean all
 
+# various hacks to make sure we use openjdk 11
+ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-11.0.12.0.7-0.el8_4.x86_64
+RUN alternatives --set java /usr/lib/jvm/java-11-openjdk-11.0.12.0.7-0.el8_4.x86_64/bin/java
```

Comment:
I am a bit concerned this might break if the upstream repositories change, but I don't know of a better way of doing this.

Reply:
Is it possible to specify a more specific package than `java-11-openjdk-devel` to ensure we get stable paths here?

Reply:
I'm pretty sure `java-11-openjdk-devel` only has one package available at a time. However, I think I've found a way to use less specific paths. Testing...